lib/Workflow/Persister.pm

```diff
@@ -53,11 +53,14 @@ sub assign_generators {
 
 sub init_random_generators {
     my ( $self, $params ) = @_;
-    my $length = $params->{id_length} || DEFAULT_ID_LENGTH;
-    eval { require Workflow::Persister::RandomId };
-    if (my $msg = $EVAL_ERROR) {
+    my $length  = $params->{id_length} || DEFAULT_ID_LENGTH;
+    try {
+        require Workflow::Persister::RandomId;
```

Comment:
Should we use Module::Runtime::require_module ?

REF: https://metacpan.org/pod/Module::Runtime#require_module(NAME)

Reply:
we could. There's a big difference between `$mod = "some_module"; require $mod;` and `require some_module;` (that is, the string argument versus the bareword argument.)

the behaviour of the string argument (which takes a filename only), really is the reason we need `require_module`. if require took a module name as a string argument, we wouldn't need `Module::Runtime`.

Reply:
Ok, we stick to you proposed change for now and we can add more syntatic sugar later